welcome/index.adoc

```diff
@@ -40,14 +40,14 @@ endif::[]
 
 ifdef::openshift-telco[]
 [.lead]
-This documentation describes the telco core and telco RAN DU reference design specifications (RDS).
 
 [IMPORTANT]
 ====
-This content is shared with Red Hat telco partners only.
+The telco core and telco RAN DU reference design specifications (RDS) are no longer published at this location.
+
+For the latest version of the telco RDS, see link:https://docs.openshift.com/container-platform/4.15/telco_ref_design_specs/telco-ref-design-specs-overview.html[Telco core and RAN DU reference design specifications].
```

Comment:
Ah, yes, this link does not render for me:

![Screenshot from 2024-07-01 14-50-34](https://github.com/openshift/openshift-docs/assets/57954076/1679b4cd-f268-4619-ab1f-0def22efc761)